packages/@okta/vuepress-site/docs/release-notes/2019-09-0/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2019.09.0
+
+| Change                                                                                                        | Expected in Preview Orgs |
+|---------------------------------------------------------------------------------------------------------------|--------------------------|
+| [Features API is Early Access EA in Preview and Production](#features-api-is-ea)                              | September 4, 2019        |
+| [Mappings API is now Generally Available (GA) in Production](#mappings-api-is-now-ga-in-production)           | September 4, 2019        |
+| [Error Object in SAML Assertion Inline Hook](#error-object-in-saml-assertion-inline-hook)                     | September 4, 2019        |
+| [Rate Limits for /oauth2 Endpoints](#rate-limits-for-oauth2-endpoints)                                        | September 4, 2019        |
+| [Rate Limits for Authorization Server Public Metadata](#rate-limits-for-authorization-server-public-metadata) | September 4, 2019        |
+| [Bug Fixed in 2019.09.0](#bug-fixed-in-2019-08-0)                                                             | September 4, 2019        |
+
+### Features API is Early Access EA in Preview and Production
+
+The [Features API](/docs/reference/api/features/) provides operations to manage self-service Early Access (EA) features in your Production and Preview orgs, and Beta features in your Preview org. <!-- OKTA-241445 -->
+
+### Mappings API is now Generally Available (GA) in Production
+
+The Okta Mappings API provides operations to manage the mapping of properties between an Okta User's and an App User's
+[Profile Properties](/docs/reference/api/users/#profile-object) using [Expression Language](/docs/reference/okta-expression-language). This feature is now GA in Production. <!-- OKTA-241945 -->
+
+### Error Object in SAML Assertion Inline Hook
+
+For the SAML Assertion Inline Hook, if an external service returns an `error` object, Okta will now deny the SAML request and redirect the end user to an error page displaying the text string sent in `error.errorSummary`. <!-- OKTA-195167 -->
```

Comment:
I bolded what I edited in the sentence:

For the SAML Assertion Inline Hook, if an external service returns an `error` object, Okta **now denies** the SAML request and **redirects** the end user to an error page **that displays** the text string sent in `error.errorSummary`. <!-- OKTA-195167 -->